platform/src/abstractions/EducationBenchmarking.Platform.Functions/EducationBenchmarking.Platform.Functions.csproj

```diff
@@ -4,6 +4,7 @@
         <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
         <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+        <RuntimeIdentifiers>win-x86</RuntimeIdentifiers>
```

Comment:
Will these additions affect only Azure-deployed functions, or is it likely to break debugging on the Mac, for example?

Reply:
It's only just for publish so shouldn't break run/debug locally.

https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#runtimeidentifiers

Reply:
linux-x64 has been setup on web solution and all is still working fine for folks locally.